### ISSUE TITLE
[question] Where to put manual .el files?

### ISSUE THREAD

**Issue by poulpoulsen** [bbatsov/prelude#1116]
Hello,
i try to load a .el file but emacs doesnt find it.
I read, that prelude load and parse every .el file inside \personal\ folder.
But if i put the .el file there, emacs doesnt find it.
Then i read, that files inside .emacs.d\lisp are autoload. But with no luck.
i use Windows81_64 with emacs 25.1.2 and prelude.
The file is org-protocol-capture-html.el

Here is the error in init.el. I dont know why there, because emacs runs until the greeting message and tip of the day.
Then comes the error:

Warning (initialization): An error occurred while loading ‘c:/Users/Held/.emacs.d/init.el’:

error: Non-hex digit used for Unicode escape

To ensure normal operation, you should investigate and remove the
cause of the error in your initialization file.  Start Emacs with
the ‘--debug-init’ option to view a complete error backtrace.

here is my config:
(prelude-require-packages '(multiple-cursors org-brain all-the-icons org-wiki))
(set-face-attribute 'default nil
                    :family "Source Code Pro" :height 120)
(setq prelude-guru nil)
(setq prelude-use-smooth-scrolling t)

(tool-bar-mode 1)
(menu-bar-mode 1)

(set-face-attribute 'region t :background "#164040")
(add-to-list 'initial-frame-alist '(fullscreen . maximized))

(require 's)
(server-start)
(add-to-list 'load-path "C:\Users\Held\.emacs.d\lisp")
(require 'org-protocol)
(require 'org-protocol-capture-html)
(require 'org-wiki)
(require 'org-brain)
(require 'linum)
(require 'linum-relative)
(require 'hlinum)
(hlinum-activate)
(linum-relative-global-mode)

(display-time-mode t)

(defun set-my-margins ()
  (interactive)
  (setq left-margin-width 1)
  (setq right-margin-width 1)
  (setq left-fringe-width 5)
  (setq right-fringe-width 10)
  )

(add-hook 'text-mode-hook 'set-my-margins)
(add-hook 'prog-mode-hook 'set-my-margins)

(setq linum-format " %d ")

(add-hook 'prog-mode-hook 'linum-mode)

(setq-default tab-width 2)

(add-hook 'prog-mode-hook
	  (lambda () (setq prettify-symbols-alist
		      '(
			("=>" . ?⇒)
			("<-" . ?←)
			("->" . ?→)
			("<<<" . ?⋘)
			(">>>" . ?⋙)
			("lambda" . ?λ)
			("function" . ?λ)
			))))

(global-prettify-symbols-mode t)

Any help appreciate
Regards
Poul

**Comment by KarlFish** [comment]
Try escaping `(add-to-list 'load-path "C:\Users\Held.emacs.d\lisp")` to be `(add-to-list 'load-path "C:\\Users\\Held.emacs.d\\lisp")`

**Comment by poulpoulsen** [comment]
hello,
yes i tried it and it works. Thanks!
But what about the autoload/parse thing inside the personal folder.
Is it true? And if yes, why does this file not load?

Regards
Poul

**Comment by KarlFish** [comment]
Hmm. I did a clean install of prelude with emacs-25.2 on windows 7, then pasted the code you wrote in `personal/test.el`. I got warnings about some of the org packages not being installed, but the file was definitely loaded.

Does a buffer pop up when you open emacs with? Does `*Messages*` buffer have any warnings? 

Try adding `(message "personal el file loaded successfully")` to the bottom of the file, does that text appear in `*Messages*` buffer?

I don't have any good ideas of what's causing this issue.
